### ISSUE TITLE
Integration stopped loading after core 2024.6.4

### ISSUE THREAD

**Issue by bilogic** [hristo-atanasov/Tasmota-IRHVAC#149]
> Configuration warnings
> Platform error 'climate' from integration 'tasmota_irhvac' - cannot import name 'MqttAvailability' from 'homeassistant.components.mqtt.mixins' (/usr/src/homeassistant/homeassistant/components/mqtt/mixins.py)

I'm getting this when I check the YAML config. Was working fine in 2024.5.5

Any idea?

**Comment by nao-pon** [comment]
@bilogic Is your Tasmota-IRHVAC version 2024.6.0 or higher?

**Comment by bilogic** [comment]
I updated to this repo released few days, or how do I check?

**Comment by nao-pon** [comment]
The current source code does not import 'MqttAvailability'. It may not be updated for some reason. Please try re-download using HACS.

**Comment by bilogic** [comment]
I used a git pull, do you have the instructions for "re-downloading using HACS"?

**Comment by bilogic** [comment]
hmm I see some problem, let me try again

**Comment by bilogic** [comment]
So sorry, it works now. Thank you!
